sympy/vector/implicitregion.py

```diff
@@ -210,7 +376,8 @@ def rational_parametrization(self, parameters=('t', 's'), reg_point=None):
         =========
 
         - Christoph M. Hoffmann, Conversion Methods between Parametric and
-        Implicit Curves and Surfaces.
+        Implicit Curves and Surfaces, 1990. Available:
```

Comment:
Publisher and authors are more important than the title itself. Christoph M. Hoffmann? Perdue e-Pubs?

Reply:
OK, I will add it.